numpy/core/src/umath/override.c

```diff
@@ -137,6 +137,9 @@ normalize_reduce_args(PyUFuncObject *ufunc, PyObject *args,
         return -1;
     }
 
+    static PyObject *NoValue = NULL;
```

Comment:
C90 requires this be declared at the top of the function with the other variables (which is why the test fails)